communication/website-guidelines.md

```diff
@@ -0,0 +1,33 @@
+# Website guidelines
+
+## General content
+
+The SIG-Docs [website subproject] maintains the kubernetes.io website,
+including site content, documentation, and generated reference documentation.
+SIG-Docs maintains guides for website contributors, including a [content guide]
+detailing the types of content allowed, and a [style guide].
+
+[Contribute to Kubernetes docs]
+
+## Blog 
+
+The [Kubernetes Blog] is a subproject of SIG-Docs and operated by the [blog team].
+
+[Submit a blog post]
+
+## Community statements
+
+Prominent notices on Kubernetes web sites (such as site-wide or front-page banners)
+may be used occasionally to publish statements on behalf of the Kubernetes community.
+The content of those statements must be approved by the [Kubernetes Steering Committee].
```

Comment:
I'd like to move away from using the word "statements" and instead choose another word such as announcements.
Or expand your description to include statements and announcements if you think they are different things.

nit?: s/statements/announcements/
nit, line 22: Could simplify, The content must be approved ... by opening a pull request with the proposed content? 
nit, line 23: Content in the form of a site wide banner may remain ...

Reply:
I do think statements and announcements are distinct. If there's agreement we want to treat them the same way and include statements and announcements using the banner in this policy, I can update. Thoughts from other reviewers?

Reply:
I was thinking about how we might implement this - maybe two different kinds of Markdown file and a `kind: statement` or `kind: announcement` in the front matter (not the meaning of `kind` as in the Kubernetes API, but broadly the same idea).
My guess is that announcements are _always_ time limited and are always linked to an upcoming or recent event, whereas statements might not be.

As a website reviewer, I hope it'll be obvious which changes need steering committee approval and which don't.

Reply:
I would lean towards treating them the same way. In an event where an announcement and a previously published statement may clash it should still be bubbled up as to which should take precedence in addition to steering visibility of such a change.

Reply:
Maybe we can just use `notice` instead of separating them into announcements or statements?

Reply:
+1 to @nikhita usage of term `notice`.  Per our steering discussion, I thought we intentionally did not want to treat a statement and announcement as separate concepts, and more just have a consistent practice for the banner as a whole.

Reply:
notice sounds good to me, as does removing ambiguity about what category a given banner falls into

Reply:
Updated with "notice". I don't think the mechanics need to be spelled out here, but I'd imagine banner content living in a dir in the website repo with the right groups as approvers